src/features/editor/Editor.tsx

```diff
@@ -9,10 +9,25 @@ import "ace-builds/src-min-noconflict/theme-dawn";
 import "ace-builds/src-min-noconflict/ext-language_tools.js";
 import { EditorPannel } from "./pannel/EditorPannel";
 import { useTabs } from "../../common/hooks/useTabs";
+import { Actions as EditorActions } from "./Actions";
 
-export function Editor() {
+import "./styles.scss";
+import { useQLDB } from "../../common/hooks/useQLDB";
+import { TransactionExecutor } from "amazon-qldb-driver-js/dist/src/TransactionExecutor";
+import { dom } from "ion-js";
+
+export function Editor(): JSX.Element {
   const [theme] = useContext(ThemeContext);
   const [tabs, content, setTabContent] = useTabs();
+  const { results, error, query, clear } = useQLDB<dom.Value>(
+    "TestLedger",
```

Comment:
This will be made dynamic in the next PR where I'll build the left pannel that lists ledgers and lets the user selects it